test/Writer/StandaloneExtensionManagerTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+namespace ZendTest\Feed\Writer;
+
+use PHPUnit\Framework\TestCase;
+use Zend\Feed\Reader\StandaloneExtensionManager;
```

Comment:
This is supposed to be Writer

Reply:
Ah, my IDE had collapsed the use statements so I didn't see them. Fixed.